frontend/src/libs/components/image/image.tsx

```diff
@@ -0,0 +1,29 @@
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  src: string;
+  alt: string;
+  className?: string;
+  width?: string;
+  height?: string;
+};
+
+const Image: React.FC<Properties> = ({
```

Comment:
If it's common ui component it should be witdth 100%

Reply:
right